jitsi_meet/android/src/main/kotlin/com/gunschu/jitsi_meet/NativeViewFactory.kt

```diff
@@ -8,8 +8,8 @@ import io.flutter.plugin.platform.PlatformView
 import io.flutter.plugin.platform.PlatformViewFactory
 
 class NativeViewFactory(val activity: Activity) : PlatformViewFactory(StandardMessageCodec.INSTANCE) {
-    override fun create(context: Context, viewId: Int, args: Any?): PlatformView {
+    override fun create(context: Context?, viewId: Int, args: Any?): PlatformView {
         val creationParams = args as Map<String?, Any?>?
-        return NativeView(activity, context, viewId, creationParams)
+        return NativeView(activity, context!!, viewId, creationParams)
```

Comment:
Don't ignore the nullability of the type here. Change the `NativeView` instead to accept a nullable type.